Dockerfile

```
@@ -1,14 +1,23 @@
-FROM rasa/rasa_nlu:latest-full
+FROM rasa/rasa:1.9.6-full
 
+WORKDIR "/"
+
+COPY setup.py .
 COPY requirements.txt .
 COPY data /data
 COPY nlu_hyperopt /nlu_hyperopt
 COPY scripts /scripts
+COPY entrypoint.sh /entrypoint.sh
```

optional: At this point you might just do, this
```suggestion
COPY . .
```

---

ah sorry! the entrypoint no longer applies, I'll remove it.